cfpb/cfgov-refresh
# Multiselect: Add close icon to selected items on page load

## Changes

- Adds close icon SVG to selected item in multiselect for filters.

## Testing

1. pull branch and `gulp clean && gulp build`
2. Visit the blog and select some topics and authors. See that (x) SVG is present on the tags.
3. Click "Apply filters" to reload page.
4. See that tags still have (x) present and can be removed. 
5. Compare to production where the (x) is absent after the filter form is submitted.
